CHANGELOG.md

```diff
@@ -7,11 +7,13 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 
 ## [Unreleased]
 
+- [#680]: defmt-print: exit when stdin is closed
```

Comment:
Please reference the PR and not the issue:
```suggestion
- [#682]: defmt-print: exit when stdin is closed
```

Reply:
I thought the issue would be more interesting to reference as it has more infos and context… I fixed it now.